Plugin/src/main/java/xyz/kyngs/librelogin/common/config/ConfigurationKeys.java

```diff
@@ -105,6 +105,13 @@ public class ConfigurationKeys {
             ConfigurateHelper::getBoolean
     );
 
+    public static final ConfigurationKey<Integer> MAX_LOGIN_TRIES = new ConfigurationKey<>(
+            "max-login-tries",
+            3,
+            "Kick the player, if the password was incorrect more or equal times.",
+            ConfigurateHelper::getInt
+    );
+
```

Comment:
I think it would probably be better to merge this with KICK_ON_WRONG_PASSWORD. 
(Allow setting -1 here to not kick players if the password is incorrect at all, add a configuration migrator which would set this to -1 if the KICK_ON_WRONG_PASSWORD was false, 1 if otherwise.)

Reply:
But what if someone enters a TOTP, should we kick them with LoginTryListener?
In my opinion we should.

Reply:
> But what if someone enters a TOTP, should we kick them with LoginTryListener? In my opinion we should do that.

I tbh don't understand how this is relevant to the question.

Reply:
Tho to answer this, probably yes